Source/ORTS.Menu/ORTimetables.cs

```diff
@@ -137,6 +139,35 @@ public static List<TimetableInfo> GetTimetableInfo(Folder folder, Route route)
             }
             return ORTTInfo;
         }
+
+        // get weatherfiles
+        public static List<String> GetTimetableWeatherFiles(Folder folder, Route route)
+        {
+            var weatherInfo = new List<String>();
+            if (route != null)
+            {
+                var directory = System.IO.Path.Combine(route.Path, "WeatherFiles");
+
+                if (Directory.Exists(directory))
+                {
+                    foreach (var weatherFile in Directory.GetFiles(directory, "*.weather-or"))
+                    {
+                        FileInfo weatherFileInfo = new FileInfo(weatherFile);
+                        weatherInfo.Add(weatherFileInfo.Name);
```

Comment:
It might be better to add the full paths (i.e. `weatherFile`) to the list here, and only shorten to a filename when displaying in the dropdown.

Reply:
The way we do that on all the other dropdowns is by using a class to store the entry, and the `ToString()` method is overridden to provide the nice display value. It would be good to try that here too.

Reply:
I was struggling here to just show the name, and admittedly the idea of creating a special class with dedicated ToString() never crossed my mind.
Will look into this.

Reply:
Updated code with new class 'WeatherFilesInfo', which holds information on available files.